.travis.yml

```diff
@@ -45,7 +48,8 @@ script:
   # Create all relevant docker images
   - travis_fold start "docker"
   - docker build -t metorikku/spark:$SPARK_VERSION --cache-from metorikku/spark:latest -t metorikku/spark:latest --build-arg SPARK_VERSION=$SPARK_VERSION --build-arg HADOOP_VERSION=$HADOOP_VERSION -f docker/spark/Dockerfile docker/spark || travis_terminate 1;
-  - docker build -t metorikku/hive:$HIVE_VERSION --cache-from metorikku/hive:latest -t metorikku/hive:latest --build-arg HIVE_VERSION=$HIVE_VERSION -f docker/hive/Dockerfile docker/hive || travis_terminate 1;
+  - docker build -t metorikku/hive1 --cache-from metorikku/hive1 -t metorikku/hive1 --build-arg HIVE_VERSION=$HIVE1_VERSION --build-arg HUDI_HIVE1_VERSION=$HUDI_HIVE1_VERSION -f docker/hive1/Dockerfile docker/hive1 || travis_terminate 1;
+  - docker build -t metorikku/hive2 --cache-from metorikku/hive2 -t metorikku/hive2 --build-arg HIVE_VERSION=$HIVE_VERSION --build-arg HUDI_VERSION=$HUDI_VERSION -f docker/hive/Dockerfile docker/hive || travis_terminate 1;
```

Comment:
you are doing twice -t ```metorikku/hive2``` you can add a second tag with the exact version (and push it later)

Reply:
Also add a latest tag to hive2

Reply:
No need to create another repo, just metorikku/hive:latest metorikku/hive:1 metorikku/hive:1.2.2

Reply:
dont need?